### DISCUSSION TITLE
Collegamento ConBee2 e Hub Aqara (con sensore porta/finestra)

### DISCUSSION THREAD

**Discussion by officialdavide** [indomus/forum#1002] [GENERALE - Discussioni generiche]
Ciao,
è possibile predisporre questa situazione:
- Hub Aqara a cui è collegato un sensore porta/finestra, sempre "by Aqara";
- vorrei instaurare una comunicazione, via ZigBee, tra ConBee2 e Hub Aqara

Vi chiedo questo perché:
1) pilotare le luci dell'hub di Aqara, tramite la ConBee2
2) tramite la ConBee2 sapere lo stato del sensore

**Comment by snakuzzo** [comment]
Non si può

**Reply by officialdavide** [reply]
Grazie Diego!

Allora ti pongo un'altra domanda: con Home Assistant e ConBee2, in qualche modo, posso controllare l'hub di Aqara?
Magari anche senza l'ausilio della rete ZigBee.

Grazie ancora per il supporto!

**Reply by snakuzzo** [reply]
Non puoi
